org.cryptomator.Cryptomator.yaml

```
@@ -124,9 +131,14 @@ modules:
         --verbose
       - cp -R Cryptomator /app/
       - ln -s /app/Cryptomator/bin/Cryptomator /app/bin/cryptomator
+      - cp -R /app/lib/* /app/Cryptomator/lib/app/
       - install -D -m0644 -t /app/share/applications/ dist/linux/common/org.cryptomator.Cryptomator.desktop
       - install -D -m0644 -t /app/share/icons/hicolor/scalable/apps/ dist/linux/common/org.cryptomator.Cryptomator.svg
+      - install -D -m0644 -t /app/share/icons/hicolor/symbolic/apps/ dist/linux/common/org.cryptomator.Cryptomator.tray.svg
+      - install -D -m0644 -t /app/share/icons/hicolor/symbolic/apps/ dist/linux/common/org.cryptomator.Cryptomator.tray-unlocked.svg
       - install -D -m0644 -t /app/share/metainfo/ dist/linux/common/org.cryptomator.Cryptomator.metainfo.xml
+      - mv /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-symbolic.svg
+      - mv /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-unlocked.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-unlocked-symbolic.svg
```

Why not installing them to this location in the first place?

---

It's renaming them and adding `-symbolic` to the filename.

The files exist in `common` without `-symbolic` in their filenames and are copied from there. This avoids having the same files doubled in `common` just with different filenames.

---

yes but can't we copy and rename those in a single step?

```suggestion
      - install -D -m0644 -T dist/linux/common/org.cryptomator.Cryptomator.tray.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-symbolic.svg
```

---

Yep. I tried it with `-t` in the first place, but that created directories. It needs to be `-T`, as you mentioned.